### PR TITLE
feat(privatedb): Display OOM kill popup by clicking on ram

### PR DESCRIPTION
**Pull request by euhmeuh**
### Description of the Change

The amount of ram becomes a color-coded button that pops up the OOM kill details.
A warning message shows the latest OOMs.
Change the popup's buttons from "Cancel/Validate" to "Close/Order more RAM".

### CODE REVIEW

File: client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml [hunk 1]
```diff
@@ -62,7 +62,7 @@
    <translation id="privateDatabase_tab_LOGS" qtlid="62064">Logs</translation>
    <translation id="privateDatabase_tab_CONFIGURATION" qtlid="4406">Configuration</translation>
    <translation id="privateDatabase_dashboard_general_informations" qtlid="27115">Informations générales</translation>
-   <translation id="privateDatabase_configurations_select" qtlid="263481">Séléctionnez votre type de base de données :</translation>
+   <translation id="privateDatabase_configurations_select">Sélectionnez votre type de base de données :</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: No need to delete the qtlid in this case

File: client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml [hunk 2]
```diff
@@ -352,10 +352,10 @@
    <translation id="privateDatabase_change_ram" qtlid="37875">Modifier</translation>
    <translation id="privateDatabase_change_ram_title" qtlid="217106">Changer la quantité de la RAM</translation>
    <translation id="privateDatabase_order_select_ram_description" qtlid="217119">La quantité de ram sélectionnée remplacera la quantité actuelle.</translation>
-   <translation id="privateDatabase_order_select_ram_placeholder" qtlid="217132">Séléctionnez...</translation>
+   <translation id="privateDatabase_order_select_ram_placeholder">Sélectionnez...</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: No need to delete the qtlid in this case

File: client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml [hunk 3]
```diff
@@ -352,10 +352,10 @@
...
 
    <translation id="privateDatabase_RAM_add_select" qtlid="31858">Mémoire RAM</translation>
-   <translation id="privateDatabase_order_select_ram" qtlid="216807">Séléctionnez la quantité désirée.</translation>
+   <translation id="privateDatabase_order_select_ram">Sélectionnez la quantité désirée.</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: No need to delete the qtlid in this case

File: client/app/private-database/state/STATE.html [hunk 4]
```diff
@@ -29,10 +29,16 @@ <h3 class="oui-header__title" data-i18n-static="privateDatabase_dashboard_genera
...
-                                                <i class="fa fa-info-circle" aria-hidden="true"></i>
+                                            <button class="btn btn-xs" type="button"
+                                                    data-ng-class="stateCtrl.database.oom.realList.length >= stateCtrl.database.oom.nbOomError ? 'btn-danger' : (stateCtrl.database.oom.realList.length > 0 ? 'btn-warning' : 'btn-success')"
+                                                    data-ng-click="stateCtrl.goToOom()">
+                                                <span data-ng-bind="::stateCtrl.database.ram.value + i18n['unit_size_' + stateCtrl.database.ram.unit]"></span>
                                             </button>
+                                            <p class="small" data-ng-class="stateCtrl.database.oom.realList.length >= stateCtrl.database.oom.nbOomError ? 'text-danger' : (stateCtrl.database.oom.realList.length > 0 ? 'text-warning' : '')">
+                                                <i data-ng-if="stateCtrl.database.oom.realList.length > 0" class="fa fa-exclamation-triangle" aria-hidden="true"></i>
```
Comment (FredericEspiau) [inline client/app/private-database/state/STATE.html]: Don't use the `<i>` anchor. Use `<strong>` if the sentence is important and use `<em>` to stress elements that can change the meaning of a sentence

File: client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml [hunk 5]
```diff
@@ -547,8 +547,12 @@
    <translation id="privateDatabase_tabs_logs_list_no_logs" qtlid="413388">Pas de log pour le moment</translation>
 
 <!-- OOM -->
+   <translation id="privateDatabase_no_oom">Aucun dépassement mémoire à signaler.</translation>
+   <translation id="privateDatabase_warning_oom">{0} dépassements mémoire on eu lieu récemment.</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: `ont`

File: client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml [hunk 6]
```diff
@@ -547,8 +547,12 @@
...
    <translation id="privateDatabase_modale_oom_title">Dépassements de la mémoire vive</translation>
-   <translation id="privateDatabase_modale_oom_info">Dépassements de la mémoire vive survenus ces {0} derniers jours. Seuls les {1} derniers dépassements sont disponibles.</translation>
+   <translation id="privateDatabase_modale_oom_confirm_button">Commander plus de RAM</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: `mémoire` instead of `RAM` ? Depends on the context

Comment (euhmeuh) [reply client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: This button has the same behaviour as its neighbour : "Changer la quantité de la RAM" so I kept the same wording.

Comment (FredericEspiau) [reply client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml]: I understand but isn't `mémoire` more appropriate than `RAM` ?

**Pull request merged**